contract-ui/hooks/useRouteConfig.tsx

```diff
@@ -166,10 +165,12 @@ export function useContractRouteConfig(
     {
       title: "NFTs",
       path: "nfts",
-      isEnabled: extensionDetectedState({
-        contractQuery,
-        feature: ["ERC1155", "ERC721"],
-      }),
+      isEnabled: contract
+        ? extensionDetectedState({
+            contractQuery,
+            feature: ["ERC1155", "ERC721"],
+          })
+        : "disabled",
```

Comment:
wouldn't this be `"loading"` ?

Reply:
reverting this change since we ended up doing this logic on the component render instead